test/e2e/autoscale_test.go

```diff
@@ -84,15 +95,26 @@ func runAutoscaleUpCountPods(t *testing.T, class, metric string) {
 }
 
 func TestAutoscaleSustaining(t *testing.T) {
-	// When traffic increases, a knative app should scale up and sustain the scale
-	// as long as the traffic sustains, despite whether it is switching modes between
-	// normal and panic.
 	t.Parallel()
-
-	ctx := SetupSvc(t, autoscaling.KPA, autoscaling.Concurrency, containerConcurrency, targetUtilization)
-	test.EnsureTearDown(t, ctx.Clients(), ctx.Names())
-
-	AssertAutoscaleUpToNumPods(ctx, 1, 10, time.After(2*time.Minute), false /* quick */)
+	for _, algo := range []string{
+		autoscaling.MetricAggregationAlgorithmLinear,
+		autoscaling.MetricAggregationAlgorithmWeightedExponential,
+	} {
+		t.Run("aggregation-"+algo, func(t *testing.T) {
+			// When traffic increases, a knative app should scale up and sustain the scale
+			// as long as the traffic sustains, despite whether it is switching modes between
+			// normal and panic.
+			t.Parallel()
+
+			ctx := SetupSvc(t, autoscaling.KPA, autoscaling.Concurrency, containerConcurrency, targetUtilization,
+				rtesting.WithConfigAnnotations(map[string]string{
+					autoscaling.MetricAggregationAlgorithmKey: autoscaling.MetricAggregationAlgorithmWeightedExponential,
```

Comment:
I think you want to use `algo` here :D 
```suggestion
					autoscaling.MetricAggregationAlgorithmKey: algo,
```

Reply:
ugh :) but at least I know we ran two times with the new one with success